### DISCUSSION TITLE
Possible to completely disable WeightedEnsemble?

### DISCUSSION THREAD

**Discussion by se-jaeger** [autogluon/autogluon#1694] [Q&A] [answered]
Hey, thanks a lot for your work on this!

I want to use AutoGluon to compare different models and run HPO. Therefore, I'm not interested in using stacking or boosting and set: `auto_stack=False, num_bag_folds=0, num_stack_levels=0` (which worked as expected). My goal is to run many remote experiments and later on check which type of model works best for a given scenario. Because the environment has little disk space, I set `keep_only_best`. The problem is that most of the time `WeightedEnsemble_L2` is the best model. However, I'm explicitly interested in single models.

So my question is basically if and how it is possible to disable the training of Ensemble models? I tried `excluded_model_types=["ENS_WEIGHTED", "SIMPLE_ENS_WEIGHTED"]` but it did not worked.

Help and answers are much appreciated! - Thanks!

**Comment by Innixma** [comment, marked as answer]
Good question! Currently it isn't possible to disable the training of the weighted ensemble during fit, but you can delete it afterwards for the purposes of comparison via `.delete_models()`. I've opened an issue (#1699) tracking support for disabling the weighted ensemble entirely.

**Reply by se-jaeger** [reply]
Thanks for your answer! Too bad but nice that it is at least possible to use AutoGluon for this.

**Reply by Innixma** [reply]
Added in https://github.com/awslabs/autogluon/pull/2145, will be available in upcoming v0.6 release on Nov 15th
